src/locales/de-DE.json

```diff
@@ -76,8 +76,8 @@
     "group": "Auswahl gruppieren",
     "ungroup": "Gruppierung aufheben",
     "collaborators": "Mitarbeitende",
-    "toggleGridMode": "Gitterlinien ein-/ausschalten",
-    "toggleStats": "Statistiken für Nerds ein-/ausschalten",
+    "toggleGridMode": "Gitterlinien",
+    "toggleStats": "Statistiken für Nerds",
```

Comment:
ignore updates in translations. they are happening automatically.

Reply:
Once this PR is merged.. the translations will appear here: https://github.com/excalidraw/excalidraw/pull/2620